migration/v9/model/assessment.go

```diff
@@ -0,0 +1,54 @@
+package model
+
+type Questionnaire struct {
+	Model
+	Name         string
+	Description  string
+	Revision     uint
+	Required     bool
+	Sections     JSON
+	Thresholds   JSON
+	RiskMessages JSON
+}
+
+type Section struct {
+	Name      string
+	Order     uint
+	Questions []Question
+}
+
+type Question struct {
+	Text        string
+	Explanation string
+	IncludeFor  []CategorizedTag
+	ExcludeFor  []CategorizedTag
+}
+
+type Answer struct {
+	Order         string
+	Text          string
+	Risk          string
+	Rationale     string
+	Mitigation    string
+	ApplyTags     []CategorizedTag
+	AutoAnswerFor []CategorizedTag
+}
+
+type CategorizedTag struct {
+	Category string
+	Tag      string
+}
+
+type Assessment struct {
+	Model
+	ApplicationID   *uint
+	Application     *Application
+	ArchetypeID     *uint
+	Archetype       *Archetype
+	QuestionnaireID uint
+	Questionnaire   Questionnaire
+	Revision        uint
+	Sections        JSON
+	Thresholds      JSON
+	RiskMessages    JSON
```

Comment:
Do we need to add the Stakeholder & StakeholderGroup at the assessment level here since we can associate them to assessments within the wizard?